test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -98,7 +98,13 @@ function main(::Type{FT}, cl_args) where {FT}
     turbconv = NoTurbConv()
     # N_updrafts = 1
     # N_quad = 3
-    # turbconv = EDMF(FT, N_updrafts, N_quad, param_set)
+    # turbconv = EDMF(
+    #     FT,
+    #     N_updrafts,
+    #     N_quad,
+    #     param_set,
+    #     surface = NeutralDrySurfaceModel{FT}(param_set),
+    # )
```

Comment:
Don't we want this uncommented?

Reply:
No, the driver should be grid-mean only, anelastic1d for now. When the coupled EDMF works we can split this into two separate tests.

Reply:
The new behavior has been tested locally and works